_data/games.yml

```
@@ -168,4 +168,22 @@
   color: "#8977FF"
   achievementCount: 5
   releaseDate: 2025-06-10
-  description: Rotate your way to victory!
\ No newline at end of file
+  description: Rotate your way to victory!
+  
+- title: Generations
+  author: Scenic Route Software
+  url: https://play.date/games/generations
+  image: ./assets/images/games/generations.png
+  color: "#7fb76f"
+  achievementCount: 50
+  releaseDate: 2023-05-23
+  description: Too many kids in this family!
+  
+- title: You're Gonna Miss the Bus!
+  author: Scenic Route Software
+  url: https://play.date/games/ygmtb
+  image: ./assets/images/games/ygmtb.png
+  color: "#f6ab19"
+  achievementCount: 30
+  releaseDate: 2023-10-10
+  description: Henry is running late for school. Again.
```

Nit: would you mind adding a newline here just to make future game addition diffs cleaner?

```suggestion
  description: Henry is running late for school. Again.

```

---

There's no broken requirement here, but the "Generations" text in your banner images is anti-aliased, and of course won't appear that way in achievement viewer apps like Trophy Case. Most games show their banners cards in "effective bitmap" form (and YGMTB, by contrast, does just that).

Again, no issues if you want to upload as-is—I just wanted to call it out!

---

Believe it or not, I left the text in the png anti-aliased for this game because I liked how it looked once pdc converted the anti-aliased pixel to black or white - it felt more "typewritery" and old-timey to me. 

I don't mind them being anti-aliased for the website.